packages/trader/src/sass/app/modules/mt5/cfd-dashboard.scss

```diff
@@ -28,13 +28,16 @@
             flex-direction: column;
             padding: 0.4rem 0rem 0.4rem 0.2rem;
         }
+
```

Comment:
pls revert the extra spaces as it is not necessary in your PR and no point on adding them since other scss files are following the same syntax.

Reply:
sure, will do it. thanks